src/translations/en.json

```diff
@@ -6507,8 +6540,9 @@
         "menu": {
           "configure_ui": "Edit dashboard",
           "help": "Help",
-          "search": "Entity search",
+          "search_entities": "Search entities (E)",
           "assist": "Assist",
+          "assist_tooltip": "Assist (A)",
```

Comment:
The (E) and (A) will only work when keyboard shortcuts are enabled. We should check that. I think the easiest way would be to remove them from the translations and append them in code.